services/motion/builtin/move_request.go

```diff
@@ -58,7 +58,6 @@ type moveRequest struct {
 	requestType requestType
 	// geoPoseOrigin is only set if requestType == requestTypeMoveOnGlobe
 	geoPoseOrigin     *spatialmath.GeoPose
-	poseOrigin        spatialmath.Pose
```

Comment:
no need to have this anymore since we use the `absoluteFS` to directly place observed obstacles into the world frame.